lib/Dialect/EmitC/Conversion/TosaToEmitC.cpp

```diff
@@ -72,6 +81,65 @@ class CallOpConversion : public OpConversionPattern<SrcOp> {
   bool explicitOperandTypes;
 };
 
+class Conv2DOpConversion : public OpConversionPattern<tosa::Conv2DOp> {
+
+public:
+  Conv2DOpConversion(MLIRContext *ctx) : OpConversionPattern(ctx) {}
+
+private:
+  LogicalResult
+  matchAndRewrite(tosa::Conv2DOp convOp, ArrayRef<Value> operands,
+                  ConversionPatternRewriter &rewriter) const override {
+    // tosa conv2D supports adding a bias after the actual convolution. We will
+    // split the convolution with bias into two operations: the convolution (as
+    // an emitc call op) and the addition of the bias (as an tosa.add op).
+    // Therefore we remove bias from the operands of the convolution.
```

Comment:
Note: This is inconsistent with the implementation of the fully connected op which does the bias add in place I think.

Reply:
There will be a follow-up for the fully connected op.